File: index/src/test/resources/runtime-helpers/java-final/Entry.java
package probe;
public final class Entry {
  static final class Names { String value(long ignored, String suffix) { return "probe." + suffix; } }
  public static void main(String[] args) throws Exception { Class.forName(new Names().value(7L, "Used")).getDeclaredConstructor().newInstance(); }
}
class Used { public Used() { System.out.println("USED"); } }
class Unused { public Unused() { System.out.println("UNUSED"); } }

File: index/src/test/kotlin/dev/kartograph/index/fixture/helperobject/Entry.kt
package probe.helperobject
object Names { fun value(ignored: Long, suffix: String): String = "probe.helperobject." + suffix }
class Used { init { println("USED") } }
class Unused { init { println("UNUSED") } }
object Entry { @JvmStatic fun main(args: Array<String>) { Class.forName(Names.value(7L, "Used")).getDeclaredConstructor().newInstance() } }
